engine/src/main/java/org/terasology/rendering/world/RenderableWorldImpl.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2014 MovingBlocks
+ * Copyright 2016 MovingBlocks
```

Comment:
@emanuele3d, I think it's a really good idea! After this PR is merged, instead of giving WorldRenderer object within in a context to other nodes, a WorldRenderer instance can be turned into a node, as you mentioned.  For instance nodes that require `renderChunks()` method can use WorldRenderer node, maybe via `getDependentNodes().get(WorldRendererNode.class)` or also maybe a syntactic sugar achieved by annotations:
```ocaml
@In 
WorldRendererNode worldRenderer;
```
(Side note: As we discussed in the meeting, this approach has some issues but I am writing here just in case.)